forms/TabSet.php

```diff
@@ -152,17 +152,18 @@ public function push(FormField $field) {
 	 * @param string $name Name of the field to insert before
 	 */
 	public function insertBefore($insertBefore, $field) {
-		parent::insertBefore($insertBefore, $field);
 		if($field instanceof Tab) $field->setTabSet($this);
 		$this->sequentialSet = null;
+		return parent::insertBefore($insertBefore, $field);
 	}
 
 	public function insertAfter($insertAfter, $field) {
-		parent::insertAfter($insertAfter, $field);
 		if($field instanceof Tab) $field->setTabSet($this);
 		$this->sequentialSet = null;
+		return parent::insertAfter($insertAfter, $field);
 	}
-
+	
+	// TODO: What's the point in overriding this if it doesn't do anything?
 	public function removeByName( $tabName, $dataFieldOnly = false ) {
```

Comment:
It's been this way since the very first time it was added to source control.

You can remove this method.